magento/magento2
# #32910 fix rewriting category product URL for the one store group

### Description (*)

Fixed adding category to product in store view scope to generate all url rewrites

### Fixed Issues (if relevant)

1. Fixes magento/magento2#32910

### Manual testing scenarios (*)

1. Create 2 store views
2. Create category with anchor enabled (and in 2.4.2 make sure category/product url generation is enabled)
3. Add new product without category
4. Open product edit in storeview scope
5. Add product to category and save

### Contribution checklist (*)
